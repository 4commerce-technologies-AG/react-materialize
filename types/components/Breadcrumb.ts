import * as React from 'react';

declare namespace Breadcrumb {
}

declare class Breadcrumb extends React.Component {}

export = Breadcrumb;
