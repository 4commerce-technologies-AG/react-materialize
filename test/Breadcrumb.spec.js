import React from 'react';
import { shallow } from 'enzyme';
import Breadcrumb from '../src/Breadcrumb';

describe('<Breadcrumb />', () => {
  test('renders', () => {
    expect(shallow(<Breadcrumb />)).toMatchSnapshot();
  });
});
