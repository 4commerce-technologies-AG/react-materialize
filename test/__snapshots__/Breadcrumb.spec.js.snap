// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Breadcrumb /> renders 1`] = `
<nav
  className="row"
>
  <div
    className="nav-wrapper"
  >
    <Col
      s={12}
    />
  </div>
</nav>
`;
